dialect/sql/builder_test.go

```diff
@@ -2067,6 +2067,12 @@ func TestEscapePatterns(t *testing.T) {
 		Query()
 	require.Equal(t, "UPDATE `users` SET `name` = NULL WHERE `nickname` LIKE ? ESCAPE ? OR `nickname` LIKE ? ESCAPE ? OR `nickname` LIKE ? ESCAPE ? OR LOWER(`nickname`) LIKE ? ESCAPE ?", q)
 	require.Equal(t, []any{"\\%a8m\\%%", "\\", "%\\_alexsn\\_", "\\", "%\\\\pedro\\\\%", "\\", "%\\%abcd\\%efg%", "\\"}, args)
+
+	//Chinese
+	query, args := Select("*").From(Table("dataset")).
+		Where(Contains("title", "_第一")).Query()
+	require.Equal(t, "SELECT * FROM `dataset` WHERE `title` LIKE ?", query)
+	require.Equal(t, []any{"%\\_第一%"}, args)
```

Comment:
```suggestion
	q, args = Select("*").From(Table("dataset")).
		Where(Contains("title", "_第一")).Query()
	require.Equal(t, "SELECT * FROM `dataset` WHERE `title` LIKE ?", q)
	require.Equal(t, []any{"%\\_第一%"}, args)
```

Use the same variables from above to keep this function consistent. Thanks 🙏